Arthur Bates

AliasAJ opened the pull request:
first PR

almost done with array methods

lefrenk commented on src/class.js:
```diff
@@ -7,6 +7,25 @@
 // Return true if the potential password matches the `password` property.  Otherwise return false.
 
+class User {
+  constructor(options) {
```
You can use object destructuring here:

```
  constructor({ email, password }) {
    this.email = email;
    this.password = password;
  }
```

lefrenk commented on src/class.js:
```diff
@@ -7,6 +7,25 @@
+  }
+}
+
+const newUser = new User({
```
Please remove and 'test' code before your final `git push`!

lefrenk commented:
Hi Aquila! Solid work as usual.

- Tests pass ✔️
- Code is consistent with JS idioms and style ✔️
- Solid grasp of Classes, Prototype inheritance, this, and recursion ✔️

Keep up the fantastic work! Please close this PR when you get a chance. Thanks!

AliasAJ committed:
Arthur Bates

first PR

almost done with array methods

AliasAJ committed:
Second PR

still working on closure.js and array-methods.js

good on callbacks and function-conversion

TERR1E reviewed (commented):
Hi Arthur,

Wanted to share some helpful links on array methods and closures with you:

https://codeburst.io/javascript-map-vs-foreach-f38111822c0f

https://medium.com/@JeffLombardJr/understanding-foreach-map-filter-and-find-in-javascript-f91da93b9f2c

Closure & Callbacks: https://gist.github.com/amysimmons/3d228a9a57e30ec13ab1
